### DISCUSSION TITLE
any way to generate miyoogamelist.xml for only a specific rom folder from with in OnionOS?

### DISCUSSION THREAD

**Discussion by erfanCA01** [OnionUI/Onion#1433] [Q&A]
Hi,

Is there anyway to do it for a single rom folder instead of all rom folders, when selecting "Generate miyoogamelist with digest names" in Tweaks app? I ask this because I only need GBA games to have digest names, the arcade and AMIGA i made custom gamelist with skraper that should not be touched. but if onion touches it , a lot of games wont be recognized in AMIGA folder lol

**Comment by Phanturian** [comment]
I would make backups of the miyoogamelists you want to preserve, run the generate tool in Tweaks, then put the preserved miyoogamelists back. This would take very little time and effort, plus it's good to make backups of your custom build.
